tools/lint-js.js

```diff
@@ -207,26 +198,24 @@ if (cluster.isMaster) {
   }
 
   function printProgress() {
-    if (!showProgress)
-      return;
+    if (!showProgress) return;
 
     // Clear line
-    outFn('\r' + ' '.repeat(lastLineLen) + '\r');
+    outFn(`\r ${' '.repeat(lastLineLen)}\r`);
 
     // Calculate and format the data for displaying
     const elapsed = process.hrtime(startTime)[0];
     const mins = padString(Math.floor(elapsed / 60), 2, '0');
     const secs = padString(elapsed % 60, 2, '0');
     const passed = padString(successes, 6, ' ');
     const failed = padString(failures, 6, ' ');
-    var pct = Math.ceil(((totalPaths - paths.length) / totalPaths) * 100);
+    var pct = Math.ceil((totalPaths - paths.length) / totalPaths * 100);
```

Comment:
I prefer the added parentheses so the order of operations is immediately clear.

Reply:
Another prettier auto-change. My mistake.